platform/darwin/scripts/generate-style-code.js

```diff
@@ -97,6 +97,16 @@ global.testArrayImplementation = function (property) {
     }
 };
 
+global.enumDoc = function (property, value) {
+    if ('docs' in property) {
+        return `    /**
+     ${property.docs[property.values.indexOf(value)]}
+     */`;
+    } else {
+        return "#warning documentation missing.";
```

Comment:
These warnings might be easier to track if they included the names of the enum values that are missing documentation.